src/action.c

```diff
@@ -557,6 +565,15 @@ actions_run(struct view *activator, struct server *server,
 					resize_edges);
 			}
 			break;
+		case ACTION_TYPE_MOVETO:
+			if (view) {
+				const char *x = get_arg_value_str(action, "x", NULL);
+				const char *y = get_arg_value_str(action, "y", NULL);
+				if (x && y) {
+					view_move(view, atoi(x), atoi(y));
+				}
```

Comment:
I think we we should add them as `int` args while parsing instead of parsing the string values when executing.

Reply:
So, add a `get_arg_value_int()` function?

Reply:
yes, sounds good

Reply:
Done